.gitignore

```diff
@@ -47,3 +47,4 @@ npm-debug.log
 uploads/
 test_uploads/
 *.mo
+assets/language_options.json
```

Comment:
I think this file should go under `static/locale/` perhaps; `assets/` is intended for just (and also intended to be moved under `static/` eventually anyway).